heat/core/dndarray.py

```diff
@@ -1361,35 +1462,53 @@ def __getitem__(self, key):
                         arr = self.__array[key]
                         gout = list(arr.shape)
                 else:
-                    warnings.warn("This process (rank: {}) is without data after slicing, running the .balance_() function is recommended".format(
-                        self.comm.rank), ResourceWarning)
+                    warnings.warn(
+                        "This process (rank: {}) is without data after slicing, running the .balance_() function is recommended".format(
+                            self.comm.rank
+                        ),
+                        ResourceWarning,
+                    )
                     # arr is empty and gout is zeros
 
-            elif isinstance(key, (tuple, list)):  # multi-argument gets are passed as tuples by python
+            elif isinstance(
+                key, (tuple, list)
+            ):  # multi-argument gets are passed as tuples by python
                 gout = [0] * len(self.gshape)
                 # handle the dimensional reduction for integers
                 ints = sum([isinstance(it, int) for it in key])
-                gout = gout[:len(gout) - ints]
+                gout = gout[: len(gout) - ints]
 
                 if self.split >= len(gout):
                     new_split = len(gout) - 1 if len(gout) - 1 > 0 else 0
                 else:
                     new_split = self.split
 
-                if isinstance(key[self.split], slice):  # if a slice is given in the split direction
+                if isinstance(
+                    key[self.split], slice
+                ):  # if a slice is given in the split direction
                     # below allows for the split given to contain Nones
                     key_stop = key[self.split].stop
                     if key_stop is not None and key_stop < 0:
                         key_stop = self.gshape[self.split] + key[self.split].stop
-                    key_set = set(range(key[self.split].start if key[self.split].start is not None else 0,
-                                        key_stop if key_stop is not None else self.gshape[self.split],
-                                        key[self.split].step if key[self.split].step else 1))
+                    key_set = set(
+                        range(
+                            key[self.split].start
+                            if key[self.split].start is not None
+                            else 0,
+                            key_stop
+                            if key_stop is not None
+                            else self.gshape[self.split],
+                            key[self.split].step if key[self.split].step else 1,
```

Comment:
This seems very inconsistent. The last ternary expression is single-lines, while the two before are multi-line

Reply:
It probably has to do with [How Black wraps lines](https://github.com/psf/black#how-black-wraps-lines).